content/en/data_jobs/databricks.md

```diff
@@ -57,29 +61,41 @@ You can choose to install the Agent globally, or on a specific Databricks cluste
    - Under **Init script path**, enter the path to your init script.
    - Click **Add**.
 
+#### Add your Datadog API key in Databricks
+
+1. In Databricks, on the cluster configuration page, click the **Advanced options** toggle.
+1. At the bottom of the page, go to the **Spark** tab.
+   {{< img src="data_jobs/databricks/configure-databricks-cluster-init-script.png" alt="Databricks UI, cluster configuration advanced options, Spark tab. A textbox titled 'Environment variables' contains values for DD_API_KEY and DD_SITE." style="width:100%;" >}}
+
+   In the **Environment variables** textbox, provide the values for the init script parameters.
+1. Click **Confirm**.
+
 {{% /tab %}}
 
 {{< /tabs >}}
 
-### Add your Datadog API key in Databricks
-1. Find your [Datadog API key][3].
-1. In Databricks, on the cluster configuration page, click the **Advanced options** toggle.
-1. At the bottom of the page, go to the **Spark** tab.
-   {{< img src="data_jobs/databricks/configure-databricks-spark-envvars-updated.png" alt="Databricks UI, cluster configuration advanced options, Spark tab. A textbox titled 'Environment variables' contains values for DD_API_KEY and DD_SITE." style="width:100%;" >}}
+### Init script parameters
```

Comment:
I don't like that this is a completely different section from the init script params section above in each setup option. I think we should just list these in context there so users can easily see all the options they should set.

Reply:
Move it to the section above